server/backend/forum/migrations/0002_initial.py

```
@@ -0,0 +1,65 @@
+            name="post",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="votes",
+                to="forum.post",
+            ),
+        ),
+        migrations.AddField(
+            model_name="vote",
+            name="user",
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)